src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0);
+}
+
+.logo {
+  margin-left: 50px;
+  margin-right: auto;
+  padding: 10px 0;
+  height: 40px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: inline-block;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  top: 56px;
+  right: 0;
+  left: 0;
+  width: 37px;
```

```suggestion
  bottom: 0;
  right: 0;
  left: 0;
  width: 100%;
```

Bottom: 0 should work as well,
Also don't hardcode width here, because this bottom line should fit each navigation item